src/style.css

```diff
@@ -87,6 +87,7 @@ code.uiContainer {
 
 .dropdown-menu {
   min-width: 300px !important;
+  z-index: 1100 !important;
```

Comment:
Does it work without !important?  Can be nice to not use !important if some other rule needs to override this for some reason in the future.

Reply:
Good point. It does work without important for min-width, but not for z-index. So I adjusted accordingly 😸